backend/plugin/db/driver.go

```diff
@@ -143,14 +143,18 @@ type MigrationInfo struct {
 	ReleaseVersion string
 	Version        string
 	Namespace      string
+	InstanceID     int
 	Database       string
+	DatabaseID     *int
 	Environment    string
 	Source         MigrationSource
 	Type           MigrationType
 	Status         MigrationStatus
 	Description    string
 	Creator        string
+	CreatorID      int
 	IssueID        string
+	IssueIDInt     *int
 	// Payload contains JSON-encoded string of VCS push event if the migration is triggered by a VCS push event.
```

Comment:
DatabaseID, CreatorID, IssueIDInt would supersedes their counterparts Namespace, Creator, IssueID should we finish the migration.